### ISSUE TITLE
Fix website accessibility issues (`Pa11y`)

### ISSUE THREAD

**Issue by antgamdia** [vmware-tanzu/kubeapps#4844]
I've run `Pa11y`  (`npx pa11y https://kubeapps.dev/`) to detect any accessibility issues on our website. 

Related to #3953 

````
Welcome to Pa11y

 > Running Pa11y on URL https://kubeapps.dev/

Results for URL: https://kubeapps.dev/

 • Error: Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
   ├── html > body > header > div > ul > li:nth-child(5) > a > img
   └── <img src="/img/slack.png">

 • Error: Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
   ├── html > body > header > div > ul > li:nth-child(6) > a > img
   └── <img src="/img/github.svg">

 • Error: Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
   ├── #mobile-menu > div > a:nth-child(1) > img
   └── <img src="/img/twitter.png">

 • Error: Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
   ├── #mobile-menu > div > a:nth-child(2) > img
   └── <img src="/img/github.svg">

 • Error: Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
   ├── #mobile-menu > div > a:nth-child(3) > img
   └── <img src="/img/slack.png">

 • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.37:1. Recommendation:  change text colour to #007eb3.
   ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
   ├── html > body > main > div:nth-child(1) > div > div > div > a
   └── <a class="button" href="/docs/latest/tutorials/getting-started/">Getting started</a>

 • Error: Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
   ├── html > body > main > div:nth-child(2) > div > div > div:nth-child(1) > img
   └── <img src="/img/simple.svg">

 • Error: Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
   ├── html > body > main > div:nth-child(2) > div > div > div:nth-child(2) > img
   └── <img src="/img/frictionless.svg">

 • Error: Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
   ├── html > body > main > div:nth-child(2) > div > div > div:nth-child(3) > img
   └── <img src="/img/seamless.svg">

 • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.01:1. Recommendation:  change text colour to #0076a8.
   ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
   ├── html > body > main > div:nth-child(3) > div > div > p:nth-child(2) > a
   └── <a aria-label="Get the ICS calendar invitation" target="_blank" rel="noopener" href="https://kubernetes.slack.com/files/U91HAPE5R/F01EGHESH4L/kubeapps-iteration-planning.ics">the invitation</a>

 • Error: Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
   ├── html > body > main > div:nth-child(4) > div:nth-child(4) > div:nth-child(1) > img
   └── <img src="/img/administration.svg">

 • Error: Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
   ├── html > body > main > div:nth-child(4) > div:nth-child(5) > div:nth-child(2) > img
   └── <img src="/img/authentication.svg">

 • Error: Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
   ├── html > body > main > div:nth-child(4) > div:nth-child(6) > div:nth-child(1) > img
   └── <img src="/img/security.svg">

 • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.37:1. Recommendation:  change text colour to #007eb3.
   ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
   ├── html > body > main > div:nth-child(4) > div:nth-child(6) > div:nth-child(2) > p:nth-child(3) > a
   └── <a target="_blank" rel="noopener" href="http://localhost:8080" class="button tertiary">Go to localhost:8080</a>

 • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.37:1. Recommendation:  change text colour to #007eb3.
   ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
   ├── html > body > div:nth-child(3) > div > div:nth-child(2) > a
   └── <a href="/docs/latest" class="button">Read the docs</a>

 • Error: This element has role of "button" but does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputCheckbox.Name
   ├── #hosts-list-container > li > input
   └── <input type="checkbox" class="host-box" aria-expanded="false" role="button" ot-accordion="true" aria-label="">

 • Error: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.     
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
   ├── #hosts-list-container > li > input
   └── <input type="checkbox" class="host-box" aria-expanded="false" role="button" ot-accordion="true" aria-label="">

 • Error: Heading tag found with no content. Text that is not intended as a heading should not be marked up with heading tags.
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.H42.2
   ├── #hosts-list-container > li > section > div:nth-child(1) > div:nth-child(2) > h3
   └── <h3 class="host-title"></h3>

 • Error: Heading tag found with no content. Text that is not intended as a heading should not be marked up with heading tags.
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.H42.2
   ├── #hosts-list-container > li > section > div:nth-child(1) > div:nth-child(2) > h4
   └── <h4 class="host-description"></h4>

19 Errors
````
